Czech locale maintenance and misc. maintenance

rampa3 opened the pull request:
- made sure that English and Czech line counts are same - check for new strings by this method
 - cleaned up some mess in my locale file
 - fixed mistakenly translated wiki key for Statue of Liberty in The Universe strings
 - misc. maintenance - fixed double line feed in English to single, to not throw off line count

rampa3 committed:
Czech locale and misc. maintenance

 - made sure that English and Czech line counts are same - check for new strings by this method
 - cleaned up some mess in my locale file
 - fixed mistakenly translated wiki key for Statue of Liberty in The Universe strings
 - misc. maintenance - fixed double line feed in English to single, to not throw off line count

ProbablePrime commented:
I only see English additions in the CZech file.

There's no need to duplicated un-translated keys, these are handled automatically. Please only add strings to the CZech file that you have translated, thank you.

ProbablePrime reviewed (changes requested):
As comment says

rampa3 commented:
I really should find better way to check for missing strings then – the untranslated wiki strings are there since those are not to be translated strings for The Universe, and my method to check if I don't miss strings until now was by line number against the en file. I will try find a different one...

ProbablePrime commented:
You're welcome to use your current method to look for untranslated strings.

Just wait until they are actually translated before opening a PR

rampa3 commented:
> I really should find better way to check for missing strings then – the untranslated wiki strings are there since those are not to be translated strings for The Universe, and my method to check if I don't miss strings until now was by line number against the en file. I will try find a different one...

Well the thing here is that the english strings you mentioned are the "wiki" strings that are said to NOT be translated right in this comment by Frooxius himself: https://github.com/Neos-Metaverse/NeosLocale/issues/569#issuecomment-929183470

Should I remove those then and add line breaks instead? (The wiki strings are there ONLY for line count.)

(One of the tasks done in the summary even is un-translating by mistake translated wiki string...)

Psychpsyo commented:
If this is only for your own count, couldn't you just add them locally on your end and not commit those changes?

rampa3 commented:
In theory yes, just would have to somehow automate the adding/removing – at the moment I work straight on my fork of the repo. Also manually adding/removing the lines would be tedious and waste of time just to produce a clean copy to commit. Any suggestions how to make this work without having to reformat the file every single time? (My method of checking if I have everything works ONLY when the Czech file follows exactly same formatting as English line wise, including same line break counts, etc.)

Psychpsyo commented:
> Any suggestions how to make this work without having to reformat the file every single time?

I use [Github Desktop](https://desktop.github.com/) which makes including / excluding specific lines on a commit really easy.
Apart from that, I'm not sure.

rampa3 committed:
Merge branch 'Neos-Metaverse:master' into master

rampa3 committed:
Replaced universe wiki strings with line breaks

 - replaced not to be translated The Universe wiki strings (Why they should not be translated was explained by Frooxius here: https://github.com/Neos-Metaverse/NeosLocale/issues/569#issuecomment-929183470) with linebreaks to remove English text trough the Czech locale document while keeping same line count (line count used to check if there are translations missing against the English locale)

rampa3 committed:
Merge branch 'master' of https://github.com/rampa3/NeosLocale

rampa3 commented:
I removed all not to be translated The Universe wiki (Froox's explanation link posted in one of the above messages) keys I used as padding to just leave those to load defaults and used line breaks to pad the lines instead. Is this what was requested? (The English mentioned as not to be allowed to be added in the initial complaint were all The Universe wiki keys, which I put in to use as padding and "anchors" (same text chunks to look for) for document alignment side by side.)
